lib/jingle/endpoint.ex

```diff
@@ -42,7 +42,7 @@ defmodule Jingle.Endpoint do
   plug Corsica,
     origins: ~r/.*\.prx\.(?:org|dev|tech|docker)$/,
     allow_headers: ~w(Authorization),
-    allow_methods: ~w(HEAD GET),
+    allow_methods: ~w(HEAD GET PUT),
```

Comment:
Remind me ... how has this not already caused problems?

(Also, does this need to include POST/DELETE as well for frontend CRUD to work?)

Reply:
frontend hadn't been doing anything but GET yet! and -- yes, Creatives will be created + destroyed through frontend. Might as well add that now! will add.